setupgui/windows/odbcdialogparams.cpp

```diff
@@ -717,6 +717,31 @@ void FormMain_OnCommand(HWND hwnd, int id, HWND hwndCtl, UINT codeNotify)
         EnableWindow(secret_id, usingSecretsManager);
       }
       break;
+    case IDC_EDIT_FED_AUTH_MODE:
+    {
+        HWND fedAuthMode = TabCtrl_1.hTabPages[FED_AUTH_TAB - 1];
+        assert(fedAuthMode);
+
+        HWND endpoint = GetDlgItem(fedAuthMode, IDC_EDIT_IDP_ENDPOINT);
+        HWND user = GetDlgItem(fedAuthMode, IDC_EDIT_IDP_USERNAME);
+        HWND pass = GetDlgItem(fedAuthMode, IDC_EDIT_IDP_PASSWORD);
+        HWND roleArn = GetDlgItem(fedAuthMode, IDC_EDIT_IAM_ROLE_ARN);
+        HWND idpArn = GetDlgItem(fedAuthMode, IDC_EDIT_IAM_IDP_ARN);
+        HWND appId = GetDlgItem(fedAuthMode, IDC_EDIT_APP_ID);
+        assert(endpoint);
+        assert(user);
+        assert(pass);
+        assert(roleArn);
+        assert(idpArn);
+        assert(appId);
+
+        wchar_t fedMode[20];
+        ComboBox_GetText(GetDlgItem(fedAuthMode, IDC_EDIT_FED_AUTH_MODE), fedMode, sizeof(fedMode));
+
+        BOOL usingOkta = wcscmp(fedMode, L"OKTA") != 0;
+        EnableWindow(appId, usingOkta);
```

Comment:
Shouldn't this be `==`? Also, haven't touched this GUI stuff all that much, can the user manually type it in instead of using the dropdown? If so, is there a case-insensitive version of this